.gitignore

```diff
@@ -49,6 +49,7 @@ src/qt/test/moc*.cpp
 .libs
 .*.swp
 *.*~*
+*.backup
```

Comment:
I guess this would not hurt, though I think we should cleanup the backup file (unlink them) in `wallet_test.cpp`.